docs/install.rst

```diff
@@ -76,22 +76,21 @@ Finally, install ``stingray`` itself::
 Test Suite
 ----------
 
-Please be sure to run the test suite before you use the package, and please report anything
```

Comment:
I think this introduction can stay

Reply:
Yes sure.

Reply:
Thanks for the feedback, I have preserved this line in commit c279353